projects/resources-and-entities.md

```diff
@@ -0,0 +1,122 @@
+- A backward-compatible extension of the concept of the Resource that seamlessly integrates with the concept of Entities.
+- Precise solutions that target each of the problems listed above.
+- A new entity signal, represented as events that describe how entities change over time. All other signals (logs, traces, metrics) will be correlated with entity signals, allowing to answer new interesting questions.
+- An [extension of OTLP](https://docs.google.com/document/d/1VUdBRInLEhO_0ABAoiLEssB1CQO_IcD5zDnaMEha42w/edit#heading=h.dovorfw3l4sf) retaining full interoperability with existing senders and receivers. OTLP will allow recording extended resources and the new entity signal.
+- A definition and implementation of new behaviors for the Collector that leverage newly available entity information:
+  - To produce information about entities it observes.
+  - To [enrich telemetry](https://docs.google.com/document/d/1VUdBRInLEhO_0ABAoiLEssB1CQO_IcD5zDnaMEha42w/edit#heading=h.ij8yjheo645z) that passes through it by additional entity-related information it possesses.
+- A set of core Entities and stable semantic conventions for those entities
+  - To bootstrap consistent identity for OTEl users: Service, Process, Host, Container and k8s.
+  - A set of prototype resource/entity detection code in multiple languages.
+
+A set of prototypes has been implemented that demonstrate what these deliverables look like in practice:
+
+### Go SDK Prototype:
+
+See https://github.com/tigrannajaryan/opentelemetry-go/pull/244
+
+### Collector Prototype
+
+See https://github.com/tigrannajaryan/opentelemetry-collector/pull/4
+
+### Collector Contrib Prototype
+
+See https://github.com/tigrannajaryan/opentelemetry-collector-contrib/pull/1/files
+
+### OTLP Protocol Buffer changes
+
+See: https://github.com/tigrannajaryan/opentelemetry-proto/pull/2/files
+
+## Staffing / Help Wanted
+
+The project has 2 TC sponsors: Tigran Najaryan and Josh Suereth. The sponsors will also actively lead and contribute to the project. We are looking for other contributors that have expertise and interest in the project.
+
+## Required staffing
+
+Project leads and TC sponsors: Tigran Najaryan and Josh Suereth
+
+Engineers willing to write prototypes and contribute:
+
+- Go - Tigran Najaryan, Tyler Yahn (maintainer)
+- Java - Josh Suereth, Jack Berg (maintainer)
+- Rust - Josh Suereth
+- Collector - Tigran Najaryan, Bogdan Drutu (maintainer), Dmitrii Anoshin (maintainer).
+
+## Meeting Times
+Once a project is started, the working group should meet regularly for discussion. These meeting times should be posted on the OpenTelemetry public calendar.
+
+## Timeline
+
+We expect the SIG to start working immediately. The timeline for specific milestones and deliverables is to be defined by the SIG.
```

Comment:
Can the project leads please start with a deadline for delivering the initial design proposal?

Reply:
Added.